lexico/cli.py

```diff
@@ -42,6 +47,8 @@ def init():
     # Step 05: Check if DB_FILE exists
     # Step 06: If not, then create necessary tables.
 
+    p = subprocess.Popen(["display", "static/logo.png"])
```

Comment:
Can we do without spawning a subprocess here?

Reply:
We can use `imagemagick` or `PIL` library but the result will be the same as a `subprocess`. The image will be popped up and closed when the `init` is done. Should I go ahead with any of these libraries?
Or do you want the image to be displayed in the command line, I mean the image to be embedded in the CLI? I didn't find any library for this.

Reply:
Let's not use this for now. We will add ASCII art later.